backend/src/middleware/slugify/uniqueSlug.spec.js

```diff
@@ -15,4 +15,11 @@ describe('uniqueSlug', () => {
       .mockResolvedValueOnce(true)
     expect(uniqueSlug(string, isUnique)).resolves.toEqual('hello-world-1')
   })
+
+  it('slugify null string', () => {
+    const string = null
+    const isUnique = jest.fn()
+      .mockResolvedValue(true)
+    expect(uniqueSlug(string, isUnique)).resolves.toEqual('anonymous')
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:40:01Z" title="Thursday, May 2nd 2019, 7:40:01 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -15,4 +15,11 @@ describe('uniqueSlug', () => {
       .mockResolvedValueOnce(true)
     expect(uniqueSlug(string, isUnique)).resolves.toEqual('hello-world-1')
   })
+
+  it('slugify null string', () => {
+    const string = null
+    const isUnique = jest.fn()
+      .mockResolvedValue(true)
+    expect(uniqueSlug(string, isUnique)).resolves.toEqual('anonymous')
```


```suggestion
    expect(uniqueSlug(string, isUnique)).resolves.toEqual('Anonymous')
```

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-06T10:56:26Z" title="Monday, May 6th 2019, 12:56:26 pm +02:00">May 6, 2019</time>_

---


Greg said slugs are lower case